tools/importer/src/main/java/com/vesoft/nebula/tools/GeoImporter.java

```diff
@@ -91,7 +91,7 @@ public void runMultiJob(Options options) throws Exception {
         this.hostAndPorts = options.getHostPort();
 
         readContent();
-        FileWriter fileWriter = new FileWriter(options.errorPath.toFile());
+        FileWriter fileWriter = new FileWriter(options.errorPath.toFile(), true);
```

Comment:
`FileWriter(String fileName, boolean append)` when the file have existed we should not append to the end  ?

Reply:
We will not run if the file exist.